DEVELOPMENT.md

```diff
@@ -4,7 +4,7 @@
 
 ### Go
 
-This repo is written in the [Go](https://golang.org) programming language, which can be installed from their [installation page](https://golang.org/doc/install). The version it was developed with was v1.16, which is the minimum required version for this SDK.
+This repo is written in the [Go](https://golang.org) programming language, which can be installed from their [installation page](https://golang.org/doc/install). The version it was developed with was v1.17, which is the minimum required version for this SDK.
```

Comment:
What are we using that's 1.17-specific that requires us to exclude 1.16 users? That's pretty uncommon in go. Historically, only big departures like the go-module system cause these sorts of breaks.